frontend/src/store/facilityBooking/action.ts

```diff
@@ -169,6 +169,12 @@ export const editBookingName = (newBookingName: string) => (dispatch: Dispatch<A
   dispatch({ type: FACILITY_ACTIONS.SET_BOOKING_NAME, newBookingName: newBookingName })
 }
 
+/**
+ * Verifies if the ending datetime is equal to or before the given start datetime
+ * @param toDate (Date) given ending datetime
+ * @param fromdate (Date) given starting datetime
+ * @returns sets the create booking error to the relevant error if any, otherwise sets empty string
```

Comment:
do write out the variable name ya
```suggestion
 * @returns sets `createBookingError` to the relevant error if any, otherwise sets empty string
```